dashboard/datasource.py

```diff
@@ -37,39 +39,39 @@ def func():
 
 class DataSource:
     def search(self, search_term: str) -> "DataSource":
-        raise NotImplementedError()
+        raise NotImplementedError()  # pragma: no cover
```

Comment:
What's the issue here?

Reply:
There's no issue I think.
This is an interface which doesn't implement these methods, so we don't have test coverage here.
Children classes like MockDataSource implement that and have coverage but the interface doesn't.

Reply:
I was sleepy. I thought it was `# noqa` :sweat_smile: 

Thanks!

Reply:
No worries 😁
I'll bump coverage to 100. Now it's 99.